### PR TITLE
Updates for script configuration

### PR DESCRIPTION
**Pull request by mizzioisf**
Updated documentation according to https://github.com/informatici/openhospital-gui/pull/1082
Should be the final update for 1.11.2

### CODE REVIEW

File: doc_admin/AdminManual.adoc [hunk 1]
```diff
@@ -583,7 +577,7 @@ The properties files can be generated automatically from the packaged _.dist_ fi
 
 NOTE: Any change to these files requires an application restart to apply the modified settings.
 
-NOTE: The properties files are automatically generated/overwritten at any Open Hospital startup, unless MANUAL_CONFIG option is set to "on" in the **oh.sh / oh.ps1** scripts.
+NOTE: The properties files can be automatically generated/overwritten at any Open Hospital startup, by setting GENERATE_CONFIG_FILES option is set to "on" in the **oh.sh / oh.ps1** scripts.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: like line 677 add "the" after "setting"

File: doc_admin/AdminManual.adoc [hunk 2]
```diff
@@ -1492,7 +1486,8 @@ dicom.storage.filesystem=Z:/OH/Shared/Dicom_images
 dicom.max.size=4M
 ----
 
-NOTE: The _dicom.properties_ file is automatically generated/overwritten at any **oh.sh / oh.ps1**  startup, unless MANUAL_CONFIG option is set to "on" in the scripts.
+NOTE: The _dicom.properties_ file can be automatically generated/overwritten at any **oh.sh / oh.ps1**  startup, by setting GENERATE_CONFIG_FILES option is set to "on" in the **oh.sh / oh.ps1** scripts.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: I would add "the" after "setting" here too

File: doc_admin/AdminManual.adoc [hunk 3]
```diff
@@ -1655,7 +1650,8 @@ log4j.category.org.isf=INFO,RollingFile,StdOut
 If the DB appender is specified, the configuration must be set to match the settings in the _database.properties_ file
 (see <<database-properties,database.properties>>).
 
-NOTE: The _log4j.properties_ file is automatically generated/overwritten at any Open Hospital startup, unless MANUAL_CONFIG option is set to "on" in the **oh.sh / oh.ps1** scripts.
+NOTE: The _log4j.properties_ file can be automatically generated/overwritten at any Open Hospital startup, by setting GENERATE_CONFIG_FILES option is set to "on" in the **oh.sh / oh.ps1** scripts.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: and finally here too

Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: Done. thanks !

File: doc_admin/AdminManual.adoc (hunk 2 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: Done. thanks !

File: doc_admin/AdminManual.adoc (hunk 1 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: Done. thanks !

**Review by mwithi — APPROVED** [review]
LGTM.

**Pull request merged**